src/react/components/pages/editorPage/editorPage.tsx

```diff
@@ -57,14 +60,20 @@ export interface IEditorPageState {
     selectionMode: SelectionMode;
     /** The selected asset for the primary editing experience */
     selectedAsset?: IAssetMetadata;
+    /** Currently selected region on current asset */
+    selectedRegions?: IRegion[];
     /** The child assets used for nest asset typs */
     childAssets?: IAsset[];
     /** Additional settings for asset previews */
     additionalSettings?: IAssetPreviewSettings;
     /** Most recently selected tag */
     selectedTag: string;
+    /** Editing tag ref */
+    editingTagRef: TagInputItem;
     /** Tags locked for region labeling */
     lockedTags: string[];
+    /** Show color picker for editing tags */
```

Comment:
Still in use? editor page shouldn't need to care about whether the color picker is showing or not.

Reply:
You're right, unused